### ISSUE TITLE
[🚀 Feature]: Chromium Devtools Accessibility.pdl has been updated, Selenium DevTools Accessibility.pdl still looks old

### ISSUE THREAD

**Issue by wat4me** [SeleniumHQ/selenium#16479]
### Description

https://chromium-review.googlesource.com/c/chromium/src/+/6935478/3/third_party/blink/public/devtools_protocol/domains/Accessibility.pdl

In this commit, Accessibility.pdl was updated, but Selenium Accessibility.pdl doesn't seem to have been updated yet. Will it be updated, or am I missing something?

**Comment by diemol** [comment]
That commit is part of Chrome 142 https://github.com/chromium/chromium/blob/142.0.7444.28/third_party/blink/public/devtools_protocol/domains/Accessibility.pdl, which has not been released yet. When it is out, the pdl used in Selenium will be updated.
